opentaskspal/src/main/java/org/dmfs/opentaskstestpal/InstanceTestData.java

```diff
@@ -79,7 +80,14 @@ public InstanceTestData(@NonNull Optional<DateTime> instanceStart, @NonNull Opti
                 .withValue(TaskContract.Instances.INSTANCE_DURATION,
```

Comment:
Is the `Instances.INSTANCE_DURATION` column need at all if it is always `DUE` which is used?

Reply:
`Instances.INSTANCE_DURATION` was added for two reasons:

* provide an easy way to the UI to get the actual absolute duration of an instance
* provide an easy way to filter instances by their duration (e.g. in an SQL query)

both could certainly achieved with `Instances.INSTANCE_START` and `Instances.INSTANCE_DUE`, but considering that these might also be `null` (which would be returned as `0` by `Cursor.getLong(int)`), it would require more work work to achieve that without a duration column.

Reply:
Oh, I realized that mistook the override in the VIEW, I though it was INSTANCE_DURATION that is nulled but that's Tasks.DURATION...  Ok, I see it then why this is needed.